pkg/apitestsv2/netportal.go

```diff
@@ -67,10 +67,10 @@ func useNetportal(inputURL string, netportalOpt *netportalOption) bool {
 	hostport := r.Host
 	ss := strings.SplitN(hostport, ":", 2)
 	host := ss[0]
-	// doesn't have k8s svc suffix, do not use nerportal
-	if !strings.HasSuffix(host, k8sServiceSuffix) {
-		return false
-	}
+	//// doesn't have k8s svc suffix, do not use nerportal
```

Comment:
test if you can access default k8s namespace

Reply:
![image](https://user-images.githubusercontent.com/28723047/154004170-26a5d683-4ad1-4585-be88-82bc578f8c03.png)

The blacklist address of the edge cluster will not be accessed, but the central cluster will be accessed